NAMESPACE

```diff
@@ -2,6 +2,7 @@
 
 export(Gargle2.0)
 export(GceToken)
+export(build_request)
```

Comment:
Would it be better to have `request_build()`, `request_develop()`, `request_make()` ?

Reply:
It seems so obvious now that you say it. Yes.

Reply:
done